arch/xtensa/src/esp32s3/hardware/esp32s3_iomux.h

```diff
@@ -0,0 +1,467 @@
+/****************************************************************************
+ * arch/xtensa/src/esp32s3/hardware/esp32s3_iomux.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_XTENSA_SRC_ESP32S3_HARDWARE_ESP32S3_IOMUX_H
+#define __ARCH_XTENSA_SRC_ESP32S3_HARDWARE_ESP32S3_IOMUX_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include "esp32s3_soc.h"
+
+/****************************************************************************
+ * Pre-preprocessor Definitions
+ ****************************************************************************/
+
+/* The following are the bit fields for PERIPHS_IO_MUX_x_U registers */
+
+/* Output enable in sleep mode */
+
+#define SLP_OE     (BIT(0))
```

Comment:
minor
```suggestion
#define SLP_OE     BIT(0)
```
here and in other places